package/base-files/files/lib/upgrade/stage2

```diff
@@ -127,6 +127,16 @@ killall -9 telnetd
 killall -9 dropbear
 killall -9 ash
 
+/etc/init.d/dnsmasq stop
+/etc/init.d/dropbear stop
```

Comment:
Does it really make sense to kill this first and stop it afterwards?

Reply:
I'd agree, please remove the kill some lines below